interfaces/builtin/u2f_devices.go

```diff
@@ -138,9 +138,9 @@ var u2fDevices = []u2fDevice{
 		ProductIDPattern: "a2ac",
 	},
 	{
-		Name:             "Ledger Blue + Nano S + Nano X",
+		Name:             "Ledger Blue + Nano S + Nano X + Nano S Plus",
 		VendorIDPattern:  "2c97",
-		ProductIDPattern: "0000|0001|0004|0005|0015|1005|1015|4005|4015",
+		ProductIDPattern: "0000|0001|0004|0005|0015|1005|1015|4005|4015|5011",
```

Comment:
Hi, I'm not able to use my Nano S Plus as well on both Firefox and Chromium (installed with Snap). My Ledger shows both product Ids `5011` and `5015`, should both numbers be listed?

That's my dump from `dmesg` when I unlock my Ledger:

```
[303120.240643] usb 1-5: USB disconnect, device number 67
[303120.817217] usb 1-5: new full-speed USB device number 68 using xhci_hcd
[303120.983786] usb 1-5: New USB device found, idVendor=2c97, idProduct=5011, bcdDevice= 2.01
[303120.983802] usb 1-5: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[303120.983808] usb 1-5: Product: Nano S Plus
[303120.983813] usb 1-5: Manufacturer: Ledger
[303120.983817] usb 1-5: SerialNumber: 0001
[303120.996328] hid-generic 0003:2C97:5011.0078: hiddev1,hidraw3: USB HID v1.11 Device [Ledger Nano S Plus] on usb-0000:00:14.0-5/input0
```

And this is when I open the ethereum app from the Ledger

```
[303123.188449] usb 1-5: USB disconnect, device number 68
[303123.497319] usb 1-5: new full-speed USB device number 69 using xhci_hcd
[303123.662892] usb 1-5: New USB device found, idVendor=2c97, idProduct=5015, bcdDevice= 2.01
[303123.662907] usb 1-5: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[303123.662914] usb 1-5: Product: Nano S Plus
[303123.662919] usb 1-5: Manufacturer: Ledger
[303123.662923] usb 1-5: SerialNumber: 0001
[303123.676136] hid-generic 0003:2C97:5015.0079: hiddev1,hidraw3: USB HID v1.11 Device [Ledger Nano S Plus] on usb-0000:00:14.0-5/input0
[303123.683758] hid-generic 0003:2C97:5015.007A: hiddev2,hidraw4: USB HID v1.11 Device [Ledger Nano S Plus] on usb-0000:00:14.0-5/input1
```

Reply:
These are all ids, should they be included as well?

https://github.com/LedgerHQ/udev-rules/blob/9bf0cbbef09437024729ea34db541535a8cb3807/add_udev_rules.sh#L3-L18

Reply:
> These are all ids, should they be included as well?
> 
> https://github.com/LedgerHQ/udev-rules/blob/9bf0cbbef09437024729ea34db541535a8cb3807/add_udev_rules.sh#L3-L18

Good question. I wasn't successful in getting any useful response in ledger discord chat, or in that 'udev-rules' repo's issues section, so I quit "beating a dead horse" trying to get some clarification.

Open 'udev-rules' repo issue:

https://github.com/LedgerHQ/udev-rules/issues/20

Maybe you can also chime in on that opened issue, and it will help get the ball rolling on this?

Here is a screenshot of the chat in their discord too.

![207623831-6e8fc9cf-d6bb-4498-a038-d78ceb7eb77b](https://user-images.githubusercontent.com/1760489/223153567-b02eb405-6ca2-4b5e-83a4-982363a3a768.png)